pkg/generated/openapi/def.bzl

```diff
@@ -14,17 +14,17 @@ def openapi_library(name, tags, srcs, openapi_targets=[], vendor_targets=[]):
   )
   go_genrule(
       name = "zz_generated.openapi",
-      srcs = srcs + ["//hack/boilerplate:boilerplate.go.txt"],
+      srcs = srcs + ["//" + pkg_prefix + "hack/boilerplate:boilerplate.go.txt"],
       outs = ["zz_generated.openapi.go"],
       cmd = " ".join([
         "$(location //vendor/k8s.io/code-generator/cmd/openapi-gen)",
         "--v 1",
         "--logtostderr",
-        "--go-header-file $(location //hack/boilerplate:boilerplate.go.txt)",
+        "--go-header-file $(location //" + pkg_prefix + "hack/boilerplate:boilerplate.go.txt)",
         "--output-file-base zz_generated.openapi",
-        "--output-package k8s.io/kubernetes/pkg/generated/openapi",
-        "--input-dirs " + ",".join(["k8s.io/kubernetes/" + target for target in openapi_targets] + ["k8s.io/kubernetes/vendor/" + target for target in vendor_targets]),
-        "&& cp pkg/generated/openapi/zz_generated.openapi.go $(location :zz_generated.openapi.go)",
+        "--output-package " + output_prefix + "k8s.io/kubernetes/pkg/generated/openapi",
+        "--input-dirs " + ",".join([input_prefix + target for target in openapi_targets] + ["k8s.io/kubernetes/vendor/" + target for target in vendor_targets]),
```

Comment:
Don't you need to modify the prefix for vendor_targets as well?

Reply:
I don't think so, no.  The only reason ``vendor_targets`` needs to exist is to support k/k's vendoring of the in-repo staging areas, and that should never be true outside of k/k (i.e. there will never be vendor_targets for a non-k/k repo).

Reply:
Got it. It might be expedient to document that in a comment; my concern is that, currently, this BUILD rule is potentially partially adjusted for vendoring k/k, but that someone looking at it anew would put their own vendored targets in `vendor_targets` and be confused as to why it's not working as expected.

Reply:
I've updated it to just use the go_prefix.